ctapipe_io_nectarcam/tests/test_nectarcameventsource.py

```diff
@@ -18,7 +18,7 @@ def test_loop_over_events():
         for telid in event.r0.tels_with_data:
             assert event.index.event_id == FIRST_EVENT_NUMBER_IN_FILE + i
             n_gain = 2
-            n_camera_pixels = event.inst.subarray.tels[telid].camera.n_pixels
+            n_camera_pixels = inputfile_reader.subarray().tels[0].camera.n_pixels
```

Comment:
subarray should be a property, so this should just read `inputfile_reader.subarray.tels[0].camera.n_pixels`. 

Also: in ctapipe the tel ids start with 1 (also simtel)

Reply:
> Also: in ctapipe the tel ids start with 1 (also simtel)

When exactly to change tel_id will need to be discussed. All NectarCAM data produced so far have tel_id=0. Is there a table mapping telescopes to their ID?